### DISCUSSION TITLE
SpiralLinux Plasma: mouse is too fast  (and other problems)

### DISCUSSION THREAD

**Discussion by newbie992** [SpiralLinux/SpiralLinux-project#230] [Q&A]
I click an the "konsole" button in the panel ... and 2 instances of konsole (sometimes 3) are started.
The same for all the other start-buttons like chromium, firefox, thunderbird, file-managers (I use dolphin and nemo).

Another strange behavior of the mouse: I intend to grab a window and move it to another place on the screen, but by clicking on the window, it is changed to full-screen-view which I don't need and don't want.

Next behavior: clicking on the "x" close-button of a window, it closes the window but closes the next window too which was under the closed window.

This makes SpiralPlasma almost unusable for my workflow.
I enabled the extended mouse-settings like described here:
https://forum.kde.org/viewtopic.php?f=289&t=153755#p405711

but nothing helps. I experimented with all kinds of settings in the extended menu.
I tried 2 other mouse-devices -because in a forum someone said that maybe the mouse is damaged - but the problem is not gone.

Ok, this terrible behavior of the mouse is not exclusive to SpiralLinux (Plasma and Xfce), but I have it on all distros like EndeavourOS, Acro-Linux, Mint etc. , but on these other distros it isn't as extreme as it is on SpiralLinux.
I think it happens because SpiralLinux is way faster than all the other distros I tested.

I wonder if there is a solution for this.

2. After deleting a swap-partition and doing partitioning on sda , SpiralLinux is not booting in 13 seconds but needs about 45 seconds.
In fstab I cannot find a swap-partition, so I cannot detect the reason for the boot-problem.
In other distros it is easy. Just putting the correct new UUID for the swap-partition and they are booting in 9 to 12 seconds as usual.
Only SpiralLinux has a problem now.
Is it necessary to make a clean install ? I am sure that the problem is gone after that.

**Comment by geckolinux** [comment]
Hi there, thanks a lot for the feedback.

1. Could you please try removing these packages and rebooting? https://github.com/SpiralLinux/SpiralLinux-project/issues/142#issuecomment-1280065611
2. Please try booting without the `quiet loglevel=0 splash` options and make note of where the boot process is hanging.

**Reply by newbie992** [reply]
Thank you @geckolinux ,
I already made a clean install. ( I don't like btrfs, therefore I select ext4-format for the partition ).
As expected, the boot-time went to 13 or 14 seconds as it should be on a ssd.

About the terrible mouse-speed.  I bought a new mouse and with it the behavior is better. So it was a hardware-problem mainly.
But not completely I guess.
Therefore I followed the advice on this kde-forum-posting:
https://forum.kde.org/viewtopic.php?f=289&t=153755#p405711

`I had to do a combination of adding to /etc/X11/xorg.conf as shown above:  
Section "InputClass"
    Identifier "evdev-mouse"
    MatchIsPointer "yes"
    Driver "evdev"
EndSection`

and install another package
sudo apt install xserver-xorg-input-evdev

**Reply by newbie992** [reply]
With this combination I can see the extended mouse-options in plasma.

The link you suggested is about the touchpad, but I don't have a touchpad, I only have the mouse.
